dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/ndk/NdkCrashHandler.kt

```diff
@@ -6,9 +6,17 @@
 
 package com.datadog.android.rum.internal.ndk
 
+import com.datadog.android.core.internal.data.Writer
+import com.datadog.android.log.internal.domain.Log
+import com.datadog.android.rum.internal.domain.event.RumEvent
 import com.datadog.tools.annotation.NoOpImplementation
 
 @NoOpImplementation
 internal interface NdkCrashHandler {
-    fun handleNdkCrash()
+    fun prepareData()
+
+    fun handleNdkCrash(
+        asyncLogWriter: Writer<Log>,
+        asyncRumWriter: Writer<RumEvent>
```

Comment:
those params should be named `logWriter` and `rumWriter`. From the `NdkCrashHandler`'s point of view, they are just writers.